content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,83 @@
+  ([Issue #14039](https://github.com/istio/istio/issues/14039))
+
+## Security
+
+- **Improved** Trust Domain Validation to validate TCP traffic as well, previously only HTTP traffic was validated.
+([Issue #26224](https://github.com/istio/istio/issues/26224))
+- **Improved** Istio Gateways to allow use of source principal based authorization when the Server's TLS mode is `ISTIO_MUTUAL`.
+([Issue #25818](https://github.com/istio/istio/issues/25818))
+- **Improved** VM security. VM identity is now bootstrapped from a short-lived Kubernetes service account token. And VM's workload certificate is automatically rotated.
+ ([Issue #24554](https://github.com/istio/istio/issues/24554))
+
+## Telemetry
+
+- **Added** Prometheus metrics to istio-agent.
+ ([Issue #22825](https://github.com/istio/istio/issues/22825))
+- **Added** Metric customization with `istioctl`.
+  ([Issue #25963](https://github.com/istio/istio/issues/25963))
+- **Added** TCP Metrics and Access Logs to Stackdriver.
+ ([Issue #23134](https://github.com/istio/istio/issues/23134))
+- **Deprecated** installation of telemetry addons by `istioctl`. These will be disabled by default, and in a future release removed entirely. More information on installing these addons can be found in the [Integrations](/docs/ops/integrations/) page.
+ ([Issue #22762](https://github.com/istio/istio/issues/22762))
+- **Enabled** Prometheus [metrics merging](/docs/ops/integrations/prometheus/#option-1-metrics-merging) by default.
+ ([Issue #21366](https://github.com/istio/istio/issues/21366))
+- **Fixed** Prometheus [metrics merging](/docs/ops/integrations/prometheus/#option-1-metrics-merging) to not drop Envoy metrics during application failures.
+ ([Issue #22825](https://github.com/istio/istio/issues/22825))
+- **Fixed** Fix unexplained telemetry which affects Kiali graph. This fix increases default outbound protocol sniffing timeout to `5s`, which has impact on server first protocol like `mysql`.
+   ([Issue #24379](https://github.com/istio/istio/issues/24379))
+- **Removed** the `pilot_xds_eds_instances` and `pilot_xds_eds_all_locality_endpoints` Istiod metrics, which were not
+accurate.
+ ([Issue #25154](https://github.com/istio/istio/issues/25154))
+
+## Installation
+
+- **Added** RPM packages for running the Istio sidecar on a VM to the release.
+ ([Issue #9117](https://github.com/istio/istio/issues/9117))
+- **Added** experimental central Istiod support for [single cluster](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps) and [multi-clusters](https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane)
+- **Fixed** an issue preventing `NodePort` services from being used as the `registryServiceName` in `meshNetworks`.
+- **Improved** gateway deployments to run as non-root by default.
+ ([Issue #23379](https://github.com/istio/istio/issues/23379))
+- **Improved** standalone operator to run as non-root by default.
```

Comment:
Improved Istiod to run as non-root by default. https://github.com/istio/istio/pull/24961

The link for the standalone operator lockdown is: https://github.com/istio/istio/pull/24960

Lock down the operator security context. https://github.com/istio/istio/pull/24963